app/exporter/models/user.py

```diff
@@ -0,0 +1,16 @@
+"""
+    This is the model class for an user.
+"""
+
+from flask import abort
+
+
+class User():
```

Comment:
Initially, I stored the users in the mongo db. There was more information then. Over time, they have evolved to a single field, but yeah, its probably not necessary